src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Specific.cs

```diff
@@ -3,11 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
+using System.ComponentModel;
```

Comment:
Oops, this file is unrelated to this fix. I added it to the wrong branch. I'll remove it.